### PR TITLE
HWKAPM-794 | docker-compose multiple compose files for test framework

### PR DESCRIPTION
**Pull request by pavolloffay**
https://issues.jboss.org/browse/HWKAPM-794

Now it is possible to define multiple docker-compose.yml files in an environment section. Files are merged using docker-compose extends feature. This allows to have two(or more) compose files.

For example `tests/app/polyglot-opentracing` contains one compose.yml file which can be used without instrumentation-framework (for users/devs to test), then in test-framework there is another compose.yml which adds specific settings(volumes, env props., networks) for test itself.

Things to consider: 
1. test compose.yml has to use relative paths from directory where is first compose.yml. Therefore it is possible to point at test scenario files which are copied to `target/test-classes/testWhatever` or directly at `test/resources/testWhatEver`

### CODE REVIEW

File: tests/instrumentation-framework/src/test/java/org/hawkular/apm/tests/dockerized/InstrumentationITest.java [hunk 1]
```diff
@@ -37,6 +37,7 @@ public void testScenarios() throws IOException {
 
         // path to test/resources
         String testResourcesPath = getClass().getClassLoader().getResource(".").getPath();
+        System.out.println(testResourcesPath);
```
Comment (jpkrohling) [inline tests/instrumentation-framework/src/test/java/org/hawkular/apm/tests/dockerized/InstrumentationITest.java]: Left over?

**Comment by pavolloffay** [comment]
@jpkrohling add ports to app's compose.yml is it fine that only `order-manager` is exposed to the host?

**Comment by jpkrohling** [comment]
Yes, requests come only to the `order-manager`. The other verticles communicate with each other via the distributed event bus.

**Comment by pavolloffay** [comment]
Added readme to `opentracing-polyglot` and exposed port for `node.js` service.

**Comment by objectiser** [comment]
retest this please

**Comment by objectiser** [comment]
@pavolloffay LGTM :+1:

**Pull request merged**